YunoHost/yunohost
# Initialize folders during .deb install instead of regen conf

## The problem

Postinstall is overly complex / lengthy and among other things we are doing a bunch of mkdir which could be done during the .deb install

## Solution

Move the various mkdir / chmods to yunohost's regen-conf init (which is called during .deb install)

## PR Status

Not tested, battling with slapd to test re-running postinstall and stuff

## How to test

Re-install the .deb on a ~fresh system, run the postinstall, check that everything's okay
